source/develop/release-management/features/gluster-self-heal-monitoring.html.md

```diff
@@ -37,7 +37,7 @@ Following columns will be added to gluster_volume_bricks table.
 
   unsynced_entries  - integer - No.of unsynced entries in the brick.
 
-  unsynced_entires_history - text - History of unsynced entries in the brick. It will a list of comma separated values.
+  unsynced_entries_history - text - History of unsynced entries in the brick. It will be a list of comma separated values. By default, last 40 entries will be stored in the field and it is configurable using VDC Option 'GlusterUnSyncedEntriesHistoryLimit'
```

Comment:
I'd change alter a bit:

By default, last 40 entries will be stored in the field. The limit can be changed using engine-config option 'GlusterUnSyncedEntriesHistoryLimit'.